### DISCUSSION TITLE
this.$auth.loginWith() not working on mobile (ios)

### DISCUSSION THREAD

**Discussion by oyish** [nuxt-community/auth-module#1498] [Support Requests]
# Problem
this.$auth.loginWIth() works on pc(desktop) properly . but on mobile, it shows me NetworkError(without status code.)and also it doesn't send request. (I checked it by backend's log) but it shows me NetworkError.
i don't know why loginWith() don't work on mobile phone only. i tried to check this with other persons phone. but occurred same problem.
i already checked cors policy and firewall. 

- versions
"nuxt": "^2.15.7",
"@nuxtjs/auth-next": "^5.0.0-1624817847.21691f1",

- Source Code
```js
this.$auth
        .loginWith('local', { data })
        .then(() => {
          alert('LOGGEDIN')
        })
        .catch((e) => {
          console.log(e.message)
          alert(e)
        })
```

please help me.

**Comment by devzom** [comment]
Provide the request error / and all logs

**Reply by N0N4M3BNS** [reply]
I have the same issues when trying on a mobile side not work. In desktop browser ok. How to debug it. Because I see no error.

**Comment by N0N4M3BNS** [comment]
@suumokr Are you already fixed your problem? help me to resolve the issues.

**Comment by sleep9** [comment]
Also not working for me. Tested on Safari (mobile) and brave (mobile)
@nuxtjs/auth-next: ^5.0.0-1648802546.c9880dc
nuxt: ^2.15.8
